src/controllers/media.ts

```diff
@@ -1,20 +1,27 @@
 import { SearchRequest } from '@universalmediaserver/node-imdb-api';
 import { ParameterizedContext } from 'koa';
 import * as _ from 'lodash';
-import * as episodeParser from 'episode-parser';
+import { MovieDb } from 'moviedb-promise';
 
 import { ExternalAPIError, MediaNotFoundError, ValidationError } from '../helpers/customErrors';
 import FailedLookups, { FailedLookupsInterface } from '../models/FailedLookups';
 import MediaMetadata, { MediaMetadataInterface } from '../models/MediaMetadata';
 import SeriesMetadata, { SeriesMetadataInterface } from '../models/SeriesMetadata';
 import osAPI from '../services/opensubtitles';
-import imdbAPI from '../services/imdb-api';
 import * as externalAPIHelper from '../services/external-api-helper';
 import { mapper } from '../utils/data-mapper';
 import { OpenSubtitlesQuery } from '../services/external-api-helper';
+import SeasonMetadata, { SeasonMetadataInterface } from '../models/SeasonMetadata';
 
 export const FAILED_LOOKUP_SKIP_DAYS = 30;
 
+export const moviedb = new MovieDb(process.env.TMDB_API_KEY);
```

Comment:
Could we instantiate the class inside a service and do this check there, rather than in this controller? Like https://github.com/UniversalMediaServer/api/blob/master/src/services/imdb-api.ts#L9